gradle/versions.gradle

```diff
@@ -13,8 +13,8 @@
 
 dependencyManagement {
   dependencies {
-    dependency 'com.fasterxml.jackson.core:jackson-databind:2.14.2'
-    dependency 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.14.2'
+    dependency 'com.fasterxml.jackson.core:jackson-databind:2.15.0-rc3'
```

Comment:
Normally rather not use an RC but in this case fixing a CVE think it's worth it

Reply:
Created an issue as a reminder: https://github.com/ConsenSys/web3signer/issues/757